tests/clustergroup-normal.expected.yaml

```diff
@@ -689,75 +715,6 @@ status: {}
 # Source: clustergroup/templates/plumbing/applications.yaml
 apiVersion: argoproj.io/v1alpha1
 kind: Application
-metadata:
```

Comment:
Spurious change? I suppose this one got moved due to the above spurious changes?